src/CreateEventQueryView/CreateEventQueryView.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {connect, PromiseState} from 'react-refetch';
+import ConnectionComponent from './../Utils/ConnectionComponent.js';
+import {Col, Container, Row} from "react-grid-system";
+import {RaisedButton} from "material-ui";
+import IconSave from "material-ui/svg-icons/content/save";
+import IconCancel from "material-ui/svg-icons/navigation/cancel";
+import Header from './../Header';
+import { css } from 'aphrodite';
+import EventTypeInformation from "./EventTypeInformation";
+import EventQueryInputArea from "./EventQueryInputArea";
+import config from "../config/config";
+import AppStyles from "../AppStyles";
+
+class CreateEventQueryView extends ConnectionComponent {
+
+    static getButtons() {
+        return (<div className={css(AppStyles.textAlignCenter)}>
+            <RaisedButton
+                label="Abort"
+                icon={<IconCancel/>}
+                className={css(AppStyles.marginAllSites)}
+                secondary={true}
+            />
+            <RaisedButton
+                label="Save"
+                icon={<IconSave/>}
+                className={css(AppStyles.marginAllSites)}
+                primary={true}
+            />
+        </div>);
+    }
+
+    render() {
+        const allFetches = PromiseState.all([this.props.eventType, this.props.attributes]);
+        const eventType = this.props.eventType.value;
+        const attributes = this.props.attributes.value;
+        const connectionIncomplete = super.render(allFetches);
+        if(connectionIncomplete) {
+            return connectionIncomplete;
+        }
+        return (
+            <div>
+                <Header title={"Create Event Query for " + eventType.Name}/>
+                <Container>
+                    <Row>
+                        <Col md={4}>
+                            <EventTypeInformation attributes={attributes}/>
+                        </Col>
+                        <Col md={8}>
+                            <EventQueryInputArea/>
```

Comment:
The area is not really an area but rather a line, that's a bit irritating to be honest.
Also there is no Desription text what is supposed to be put in there so my gueass was that one should hand the timestamp parameter as it is aligned with that atribute.
Please overthik that styling.